client/src/featureform/register.py

```diff
@@ -3637,8 +3658,8 @@ def df_transformation(
                         f"DF transformation {transformation_message} requires correct inputs "
                         f" '{nv}' is not a valid tuple: {e}"
                     )
-            if inputs[i][1] == "":
-                inputs[i] = (inputs[i][0], self.__run)
+            # if inputs[i][1] == "":
+            #     inputs[i] = (inputs[i][0], self.__run)
```

Comment:
Did you meant to remove this or should you uncomment it?

Reply:
i uncommented this because I wanted sterlings thoughts on it hahah.